lua/entities/gmod_wire_expression2/core/serverinfo.lua

```diff
@@ -78,3 +78,7 @@ end
 e2function number tickInterval()
 	return engine.TickInterval()
 end
+
+e2function number tickRate()
+	return 1 / engine.AbsoluteFrameTime()
```

Comment:
The function name is too similar to tickInterval. Should be realInterval and return engine.AbsoluteFrameTime() instead of the inverse.